monitors/test/SyntheticPegMonitor.js

```diff
@@ -0,0 +1,106 @@
+const { toWei, toBN } = web3.utils;
+const winston = require("winston");
+const sinon = require("sinon");
+
+// Price feed mock.
+const { PriceFeedMock } = require("../../financial-templates-lib/test/price-feed/PriceFeedMock.js");
```

Comment:
Note that this pr ONLY uses `PriceFeedMock` to simulate both the `uniswapPriceFeed` and `medianizerPrice`. Previously this was implemented directly using an instance of `uniswapPriceFeed` and `CryptoWatchPriceFeed`. This implementation can be found [here](https://github.com/UMAprotocol/protocol/blob/fe2e53a7d2/monitors/test/SyntheticPegMonitor.js).

Is this how we want to test this logic? the reason for this refactor is that this is how we mock prices in all other contexts, like liquidator and disputer.